pkg/background/update_request_controller.go

```diff
@@ -370,6 +370,50 @@ func (c *controller) deletePolicy(obj interface{}) {
 	}
 }
 
+func (c *controller) deleteNSPolicy(obj interface{}) {
```

Comment:
It will be good if we can write unit tests for deleteNSPolicy function?

Reply:
Unit test cant be added for this function. i tried with fake client as well.